typescript/environments.d.ts

```diff
@@ -45,15 +45,26 @@ export type FormsAppEnvironmentStyles = BaseFormsAppEnvironmentStyles & {
     /** Button configuration for the Cancel Prompt dialog No button */
     cancelPromptNo?: ButtonConfiguration
   }
+  /** The icon to display on validation error messages */
+  validationIcon?: {
+    /**
+     * Must be a valid Material Icon code as it appears here:
+     * https://fonts.google.com/icons
+     */
+    icon: string
+    /** The alt text for the icon */
+    altText?: string
```

Comment:
can we go with ariaLabel or accessibleLabel, altText being for actual images.

Reply:
Yep.